hudi-common/src/main/java/org/apache/hudi/common/table/view/RocksDbBasedFileSystemView.java

```diff
@@ -167,15 +167,15 @@ protected void storePartitionView(String partitionPath, List<HoodieFileGroup> fi
     rocksDB.prefixDelete(schemaHelper.getColFamilyForView(),
         schemaHelper.getPrefixForSliceViewByPartition(partitionPath));
     rocksDB.prefixDelete(schemaHelper.getColFamilyForView(),
-        schemaHelper.getPrefixForDataFileViewByPartition(partitionPath));
+        schemaHelper.getPrefixForBaseFileViewByPartition(partitionPath));
```

Comment:
another thing to ensure.. no rocksdb level schema changes due to this . Seems fine..